### PR TITLE
Add FighterSpecializer_MewTwo

### PR DESCRIPTION
**Pull request by SuddyN**

### CODE REVIEW

File: src/cpp.rs [hunk 1]
```diff
@@ -2378,6 +2378,19 @@ pub mod root {
             }
         }
 
+        pub mod FighterSpecializer_Mewtwo {
+            #[allow(unused_imports)]
+            use super::super::super::root;
+            extern "C" {
+                #[link_name = "\u{1}_ZN3app25FighterSpecializer_Mewtwo22save_shadowball_statusERNS_21FighterModuleAccessorEbi"]
+                pub fn save_shadowball_status(
+                    arg1: *mut root::app::BattleObjectModuleAccessor,
```
Comment (WuBoytH) [inline src/cpp.rs:2387]: This should be `FighterModuleAccessor`

you can just transmute the boma pointer to this type and it'll be fine

**Review by WuBoytH — CHANGES_REQUESTED** [review]

**Review by techyCoder81 — APPROVED** [review]

File: src/cpp.rs (hunk 1 above)
Comment (SuddyN) [reply src/cpp.rs:2387]: copied from Lucario's identical function:
```
extern "C" {
    #[link_name = "\u{1}_ZN3app26FighterSpecializer_Lucario21save_aura_ball_statusERNS_26BattleObjectModuleAccessorEbi"]
    pub fn save_aura_ball_status(
        arg1: *mut root::app::BattleObjectModuleAccessor,
        arg2: bool,
        arg3: i32
    );
}
```

Comment (WuBoytH) [reply src/cpp.rs:2387]: OK but look at the symbol for Mewtwo's, it says FighterModuleAccessor for a reason

Comment (SuddyN) [reply src/cpp.rs:2387]: they're the same function :( but I did it

**Review by WuBoytH — APPROVED** [review]

File: src/cpp.rs (hunk 1 above)
Comment (WuBoytH) [reply src/cpp.rs:2387]: It's just to be accurate, just in case. Blame the intern who made it different.

**Pull request merged**